api/src/test/java/com/github/streamshub/console/kafka/systemtest/deployment/KeycloakResourceManager.java

```diff
@@ -29,26 +33,56 @@ public Map<String, String> start() {
             throw new UncheckedIOException(ioe);
         }
 
+        int port = 8443;
+        TlsHelper tls = TlsHelper.newInstance();
+        String keystorePath = "/opt/keycloak/keystore.p12";
+
         keycloak = new GenericContainer<>("quay.io/keycloak/keycloak:26.0")
```

Comment:
I'd extract a constant for the image so that it is easier to find when it is time to upgrade.

Reply:
I've been thinking about using a property in the POM, along with a dependency to a Keycloak artifact so that dependabot will bump this.

Reply:
(aside:   I'd been looking for a good way to use dependabot for image updates in [Kroxylicious](https://github.com/kroxylicious/kroxylicious/issues/1179))

Reply:
Opened https://github.com/streamshub/console/issues/1349 to track this for an upcoming fix.